packages/react/src/components/SplitButton/SplitButton.tsx

```diff
@@ -0,0 +1,204 @@
+  isFromKeyboard: boolean
+  open: boolean
+}
+
+class SplitButton extends AutoControlledComponent<WithAsProp<SplitButtonProps>, SplitButtonState> {
+  static create: Function
+
+  static displayName = 'SplitButton'
+
+  static className = 'ui-splitbutton'
+
+  static slotClassNames: SplitButtonSlotClassNames = {
+    toggleButton: `${SplitButton.className}__toggleButton`,
+  }
+
+  static propTypes = {
+    ...commonPropTypes.createCommon({
+      content: false,
+    }),
+    button: customPropTypes.itemShorthand,
+    defaultOpen: PropTypes.bool,
+    menu: PropTypes.oneOfType([
+      customPropTypes.itemShorthand,
+      PropTypes.arrayOf(customPropTypes.itemShorthandWithoutJSX),
+    ]),
+    onClick: PropTypes.func,
+    onMenuItemClick: PropTypes.func,
+    open: PropTypes.bool,
+    primary: customPropTypes.every([customPropTypes.disallow(['secondary']), PropTypes.bool]),
+    secondary: customPropTypes.every([customPropTypes.disallow(['primary']), PropTypes.bool]),
+    toggleButton: customPropTypes.itemShorthand,
+  }
+
+  static defaultProps = {
+    accessibility: splitButtonBehavior,
+    as: 'span',
+  }
+
+  static autoControlledProps = ['open']
+
+  getInitialAutoControlledState(): SplitButtonState {
+    return {
+      isFromKeyboard: false,
+      open: false,
+    }
+  }
+
+  renderComponent({
+    ElementType,
+    classes,
+    accessibility,
+    styles,
+    unhandledProps,
+  }): React.ReactNode {
+    const { button, menu, primary, secondary, toggleButton } = this.props
+
+    return (
+      <ElementType className={classes.root} {...accessibility.attributes.root} {...unhandledProps}>
+        {MenuButton.create(
+          {},
+          {
+            defaultProps: {
+              accessibility: accessibility.childBehaviors.menuButton,
+              menu,
+              styles: styles.button,
+              on: [],
+              open: this.state.open,
+              onOpenChange: (e: React.SyntheticEvent, { open }: PopupProps) => {
+                e.stopPropagation()
+                this.setState({ open })
+              },
+              trigger: Button.create(button, {
+                defaultProps: {
+                  styles: styles.button,
+                  primary,
+                  secondary,
+                },
+              }),
+            },
+            overrideProps: (predefinedProps: MenuButtonProps) => ({
+              onMenuItemClick: (e: React.SyntheticEvent, menuItemProps: MenuItemProps) => {
+                this.setState({ open: false })
+
+                _.invoke(predefinedProps, 'onMenuItemClick', e, menuItemProps)
+                _.invoke(this.props, 'onMenuItemClick', e, menuItemProps)
+              },
+              onClick: (e: React.SyntheticEvent, buttonProps: ButtonProps) => {
+                _.invoke(predefinedProps, 'onClick', e, buttonProps)
+                _.invoke(this.props, 'onClick', e, buttonProps)
+              },
+              onFocus: () => {
```

Comment:
```suggestion
              onFocus: (e: React.SyntheticEvent, buttonProps: ButtonProps) => {
                _.invoke(predefinedProps, 'onFocus', e, buttonProps)
```

Missing call of original handler?